netty/src/main/java/io/grpc/netty/ProtocolNegotiators.java

```diff
@@ -97,22 +97,12 @@ public void handlerAdded(ChannelHandlerContext ctx) throws Exception {
           }
 
           @Override
-          public void handlerRemoved(ChannelHandlerContext ctx) throws Exception {
-            // Don't care.
-          }
-
-          @Override
-          @Deprecated
-          public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws Exception {
```

Comment:
Yeah, our exception handler just does what the default handler would have done.

https://github.com/netty/netty/blob/4.1/transport/src/main/java/io/netty/channel/ChannelHandlerAdapter.java#L78

+1 to removing the method.